src/Client/GroupsClient.php

```diff
@@ -30,13 +30,15 @@ class GroupsClient extends Client implements ClientInterface
     public const ACTION_ENABLE_USER = 'enable_user';
     public const ACTION_RENAME_GROUP = 'rename';
     public const ACTION_CREATE_GROUP = 'create';
-    private Group $group;
+    public const ACTION_LIST_GROUPS = 'list';
```

Comment:
```suggestion
    private const ACTION_LIST_GROUPS = 'list';
```
As we don't want to expose new constants, right?